src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy of the array before performing operations on it. The `split` method does not mutate the original string, so you can chain other methods directly without storing the intermediate array.